GoogleForCreators/web-stories-wp
# Dashboard: Update aspect ratio to 9:16

## Summary
Updates the dashboard's preview component to use 9 : 16 ratio instead of 2 : 3 to allow for full bleed.
![Screen Shot 2020-06-17 at 11 39 00 AM](https://user-images.githubusercontent.com/10720454/84936518-37ba7580-b08f-11ea-97d2-9d5b310788b4.png)
![Screen Shot 2020-06-17 at 11 39 09 AM](https://user-images.githubusercontent.com/10720454/84936528-3ab56600-b08f-11ea-874c-1b2d6d28aa44.png)
![Screen Shot 2020-06-17 at 12 54 16 PM](https://user-images.githubusercontent.com/10720454/84944538-c2ed3880-b09a-11ea-8229-2858ee3be0e0.png)


## Relevant Technical Choices
- I referenced edit-story/components/canvas/layout to make the changes necessary to our usePagePreview util as well as components/previewPage. 
- In previewPage we now have 2 divs that wrap around the `displayElement` imported from edit-sory. They are: `PreviewSafeZone` (new bonus wrapper to control the overflow) and `FullBleedPreviewWrapper` (formerly: `previewWrapper`). The `FullBleedPreviewWrapper` identifies the total space for the preview including the full bleed (9:16 ratio). The PreviewSafeZone still relies on the 2:3 ratio and allows overflow so that the fullbleed area can be seen within the previewWrapper but not baked into the height that the unitsProvider expects to calculate the story.  
- I added a new function in `usePagePreviewSize` to calculate two widths based on the fullbleed ratio (9:16) and two thirds ratio (2:3). The 2:3 ratio is the default height. 
- I added another key to the `pageSize` that's getting passed through where ever we are previewing a story or a template called `containerHeight`. To avoid cluttering up the StoryPropTypes.size in edit-story i broke down the pageSize prop on the dashboard to only pass height and width through to the unitProvider while allowing pageSize to have the extra containerHeight key in the dashboard. 

## User-facing changes
- Ratio should be updated to 9 : 16

## Testing Instructions
- Make a story and put something in the danger zone area and see that it renders in the dashboard previews on grid
- Also see that this holds true for the different dashboard views and storybook - i think i got them all. 
- (this is a helpful link for sanely checking the aspect ratio repeatedly: https://calculateaspectratio.com/) 

---



Addresses #2557
